includes/class-kadence-blocks-prebuilt-library-rest-api.php

```diff
@@ -600,6 +607,46 @@ public function get_remaining_credits( $request ) {
 			return rest_ensure_response( $response );
 		}
 	}
+	/**
+	 * Handle the event and maybe send to Prophecy WP.
+	 */
+	public function handle_event( $request ) {
+		$event_label = $request->get_param( self::PROP_EVENT_LABEL );
+		$event_data  = $request->get_param( self::PROP_EVENT_DATA );
+		$event_data  = json_decode( $event_data, true );
+
+		$event       = '';
+		$context     = array();
+
+		switch ( $event_label ) {
+			case 'ai_wizard_started':
+				$event = 'AI Wizard Started';
+				break;
+
+			case 'ai_wizard_save':
+				$event = 'AI Wizard Save';
```

Comment:
Is this supposed to be "AI Wizard Updated"? And are we sending _all_ of the event_data each time, or only the changes?

I'm not totally sure what's passed in through `data` [here](https://github.com/stellarwp/kadence-blocks/blob/35abeb96a6d6358e1d2470b158b7cdf9587e5bf3/src/plugins/prebuilt-library/ai-wizard/hooks/use-database.js#L18).